operations/network/dns/Troubleshoot_PowerDNS.md

```diff
@@ -67,97 +76,104 @@ When troubleshooting DNS problems, it may prove helpful to increase the level of
 
 1. Restart the PowerDNS service.
 
-   ```
-   ncn-m001# kubectl -n services rollout restart deployment cray-dns-powerdns
+   ```bash
+   ncn# kubectl -n services rollout restart deployment cray-dns-powerdns
    ```
 
    Example output:
 
-   ```
+   ```text
    deployment.apps/cray-dns-powerdns restarted
    ```
 
 Refer to the external [PowerDNS documentation](https://doc.powerdns.com/authoritative/settings.html#loglevel) for more information.
 
-## Verify DNSSEC Operation
+<a name="verify-dnssec-operation"></a>
 
-### Verify Zones are Being Signed with the Zone Signing Key
+## Verify DNSSEC operation
+
+<a name="verify-zones-being-signed"></a>
+
+### Verify zones are being signed with the zone signing key
 
 Check that the required zone has a DNSKEY entry; this should match the public key portion of the zone signing key.
 
-```
-ncn-m001# kubectl -n services exec deployment/cray-dns-powerdns \
--c cray-dns-powerdns -- pdnsutil show-zone system.dev.cray.com
+```bash
+ncn# kubectl -n services exec deployment/cray-dns-powerdns -c cray-dns-powerdns -- pdnsutil show-zone system.dev.cray.com
```

Comment:
nit: please add a line notifying the user they should substitute `system.dev.cray.com` with their own URL?

Reply:
Good idea. Done!